docs/deployment/deploy_gardenlet_manually.md

```diff
@@ -0,0 +1,402 @@
+# Deploy a Gardenlet Manually
+
+Deploying a gardenlet into a Kubernetes cluster manually and registering this cluster as a seed cluster automatically is only necessary in special deployment scenarios, for example, to manage clusters in a fenced environment.
+
+Once you’ve deployed the initial gardenlet manually, you can deploy new gardenlets automatically using the initial gardenlet as a template.

```

Comment:
Are you talking about the first gardenlet deployed in an installation or about the first gardenlet behind a firewall that then creates additional seed clusters?. I would either not mention this here for simplicity reasons or reformulate to make it more understandable.
The notion of "initial gardenlet" is unclear to me here.

Reply:
@danielfoehrKn : I now understand your concern. I thought saying "manually" would make it clear that it's not done using an installation tool? 

Tried to make it a clearer. If we remove the sentence I would rather remove the link to the doc for the automatic deployment as well.

Reply:
Looks good to me now. Thanks